src/wms/src/main/java/org/geoserver/wms/clip/CroppedGridCoverage2DReader.java

```diff
@@ -0,0 +1,104 @@
+/* (c) 2019 Open Source Geospatial Foundation - all rights reserved

```

Comment:
As a curiosity, these have been actually written in 2019? Could be, just asking.
For new PRs remember to update the year to 2020.

Reply:
will do